tests/experiments/test_trainer.py

```diff
@@ -0,0 +1,45 @@
+import unittest
+
+import gym
+
+from tf2rl.algos.ddpg import DDPG
+from tf2rl.experiments.trainer import Trainer
+
+
+class TestTrainer(unittest.TestCase):
+    def test_empty_args(self):
+        """
+        Test empty args {}
+        """
+        env = gym.make("Pendulum-v0")
+        test_env = gym.make("Pendulum-v0")
+        policy = DDPG(state_shape=env.observation_space.shape,
+                      action_dim=env.action_space.high.size,
+                      gpu=-1,
+                      memory_capacity=1000,
+                      max_action=env.action_space.high[0],
+                      batch_size=32,
+                      n_warmup=10)
+        trainer = Trainer(policy, env, {}, test_env=test_env)
+        print(trainer)
```

Comment:
Why are you printing `trainer` ?

Reply:
To avoid linter error

Reply:
Could you tell me why this avoids linter error?

Reply:
https://github.com/ymd-h/tf2rl/runs/1047622916?check_suite_focus=true
```shell
F841 local variable 'trainer' is assigned to but never used
```

Reply:
Maybe there are more better solutions, but checking whether parameters are default is also problematic.
These parameters can change their default values in future, which will break unit test.

Reply:
OK. I found a solution. 

I simply do not assign to local value. (12cfa48)

Reply:
Sounds good!